### PR TITLE
UCP/RNDV: Fix mem type for rma bw lane selection with pipeline protocol

### PR DESCRIPTION
**Pull request by bureddy**
Fixes https://github.com/openucx/ucx/issues/4722

### CODE REVIEW

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 17 workers (click for details)</summary>

**Note:** the logs will be deleted after 13-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/76a9c823b9a3d6730f161884212bf9615656ce4b/_main) | :question: ABORTED
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/cb304aef2c2c3ea8000964cb7c1137c84c911394/hpc-arm-cavium-jenkins_W0) | :question: ABORTED
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/7614e6144a4760d3277e6411c8436285fdc65bfb/hpc-arm-cavium-jenkins_W2) | :question: ABORTED
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/2703e2af62cf5d1171445c493e1d14eab42212bd/hpc-arm-cavium-jenkins_W3) | :question: ABORTED
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/72077c6f2d635a215e3ced89b72d9b23e14e8ad9/hpc-test-node-new_W2) | :question: ABORTED
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/1229532dd690b3770b5d26916d36cc71bcc539b8/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/8b31e8297896af901f567322fe286fc2ed544204/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/139ada64a54fbca7b687d60245f1a73d0a9e2352/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/3a7fb1be0d8414461cf35e48ffc936feb66a4514/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/d87775ea638e7d43c1a5efabb8f054b40936c421/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/3d350b21b493d34c5109782ea04e4ab88e504b72/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/ffa0c46ecf659b9896e538d668edea4afd9a3086/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/13a3c6c147d5d0f97aa1a1fd28fa92d3efc494a6/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/ba9a3bebb7082b4babcc28a6983528f5a8e9990c/hpc-arm-hwi-jenkins_W1) | :question: UNKNOWN
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/c5ae7439084f79b3c0e488f093abe16bc62accb3/hpc-test-node-new_W0) | :question: UNKNOWN
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/35eeca19838328c82e1aa2e295f50b5ebfb2b540/hpc-test-node-new_W1) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/fe4536bdcd604afdea4d86427fb4d7ca/raw/27449cce5e2dca29b2f29d48c6caf02277c9de9f/hpc-test-node-new_W3) | :question: UNKNOWN

File: src/ucp/tag/rndv.c [hunk 1]
```diff
@@ -1317,8 +1318,14 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_rtr_handler,
                       ucp_ep_peer_name(ep), ucs_status_string(status));
         }
 
+        is_pipeline_rndv = ((!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(sreq->send.mem_type) ||
+                             (sreq->send.length != rndv_rtr_hdr->size)) &&
```
Comment (dmitrygx) [inline src/ucp/tag/rndv.c]: minor: can remove braces around 
```
!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(sreq->send.mem_type
```
Also second condition is unaligned - 1 extra whitespace at the begining of the statement.

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 13-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/1411f66e40a97c6a696925c1321024273b25a925/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/50b4ba4afeac0f6d5d0cb6e07ee254198d5fdb5f/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/c111aedf6a602898b05e4265a4f2947d2723058b/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/6009da5698f90f4cc49ae266dce1a7331939651d/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/b9ac49edb2942e90cba85277397e2af207df9264/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/d208b01458d2eb5d50a43785282269187f4e0792/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/a80dea221a3cfe8d50b67e5425d3ab9baf21078b/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/218d0c3a2ccf0232090f98621f81a527f04fcc2b/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/0524d5a3be7f6c179b1a3c3dfd122251effbff13/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/244ad5eb8f225391cda07dd6d9e27d0f1974da51/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/c7afd9af5e5df54a41d31c428eb057c27e9d2b58/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/d8cda2af3e617a74e8c119ccb6b45dc7f3dd24fc/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/a2001142136a98d74ebfa8a1bca05db7dcfdb357/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/123727be284ae24f19eb71c25d9450cf6b848f1e/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/7d75e2572d5e601e0baca43c0335d2b7bc15c00a/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/b8b781f2d09c6228c138ec2773c501c0c6ef19a0/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/d7a331bff86949c0790ff6fb2cdd10a7f3718708/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/0e15e7248a3e91dd16a65d60f1fbe45970d900e0/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/50c14917ede556d8d5fb8855fbce1b84caa5566a/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/24c242f3fb2063ba448386adf8fed57d34932cca/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/7211e4ffee90d11b86aecae1fcd30b600f505847/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/cb0b867bf48dd70c931e327134424ea43dc87982/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/8ecb3b3490a8e0c283a8da2c30a62218e1b6c97a/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/742f228c56aabac38f6e635d4dddde9660f76a1f/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/59f46d6f7b4e598dc986631c009cc529/raw/e6095a8503760a2b1dc6ae2fc948c5c39e311434/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/ucp/tag/rndv.c (hunk 1 above)
Comment (yosefe) [reply src/ucp/tag/rndv.c]: this is copy paste from prev code and braces are ok (the condition logic itself seems "hairy" but we can discuss it later)

**Pull request merged**